tests/wpt/mozilla/tests/mozilla/webgl/getFramebufferAttachmentParameter.html

```diff
@@ -0,0 +1,114 @@
+    <!doctype html>
+<meta charset="utf-8">
+<title>getFramebufferAttachmentParameter input type check (issue #20593)</title>
```

Comment:
@emilio What do we do for additional WebGL tests?

Reply:
What do you mean?

Reply:
You can upstream them to https://github.com/KhronosGroup/WebGL if you think they're useful.

Reply:
I don't know enough to decide whether they're useful there. Any opinion?